src/main/java/com/labsynch/labseer/chemclasses/rdkit/ChemStructureServiceRDKitImpl.java

```diff
@@ -268,11 +269,24 @@ public RDKitStructure getRDKitStructureFromService(String molfile) throws CmpdRe
 
 	@Override
 	public int saveStructure(String molfile, StructureType structureType, boolean checkForDupes) {
+
+		Long cdId=0L;
 		try {
 			RDKitStructure rdkitStructure = getRDKitStructureFromService(molfile);
```

Comment:
Should this be an `AbstractRDKitStructure` ? It seems like it.

Reply:
no, you can't new up an abstract class so you have to pick one of the classes that implements it.